Impossible-Agent-Suite/DangerousActivities/src/agent.ts

```diff
@@ -0,0 +1,58 @@
+import { 
+  Finding, 
+  HandleTransaction, 
+  TransactionEvent, 
+  FindingSeverity, 
+  FindingType,
+} from 'forta-agent';
+import utils, { Verifier } from './utils';
+import bscDangerous from './bsc.dangerous';
+import impossibleAddresses from './impossible.addresses';
+
+const DANGEROUS_VERIFIER: Verifier = utils.listVerifier(bscDangerous);
+const IMPOSSIBLE_VERIFIER: Verifier = utils.listVerifier(impossibleAddresses);
+
+const createFinding = (impossible: string[], dangerous: string[]) => Finding.fromObject({
+  name: "Impossible Finance interaction monitor",
+  description: "Dangerous Interaction detected",
```

Comment:
Maybe could have the description be a bit more detailed. 
```suggestion
  description: "A known dangerous address has interacted with an Impossible Finance smart contract",
```